vnext/src/Libraries/Text/TextNativeComponent.windows.js

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
@rectified95 - you'll have to teach me how to get this file added to the manifest.